### ISSUE TITLE
autocomplete mentions are not read out by screenreader JAWS

### ISSUE THREAD

**Issue by JulesAchiel** [ichord/At.js#510]
I have at.js integrated with CKEditor and after tpressng the trigger key to activate the autocomplete mentions by at.js, a screen reader like JAWS does not read out the options and does not give any indication a list of options is available. Facebook has this functionality working with JAWS.

To reproduce : 
1. download and start JAWS
2. go to http://ichord.github.io/At.js/
3. invoke the autocomplete list with : and a JAWS user should get notified of the available list opened and read out the available options and the option selected.

When integrated in CKEditor or any other editor, the same should happen.

**Comment by JulesAchiel** [comment]
Are there any plans to have at.js support screenreaders ?
